src/index.js

```diff
@@ -1,8 +1,21 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
+import firebase from 'firebase';
```

Comment:
just for our own sanity, i'm going to do imports of npm packages in one chunk and then leave some whitespace and then imports of local files/modules/etc which always get imported relatively